Tests/BackendServiceTests.swift

```diff
@@ -111,4 +116,29 @@ class BackendServiceTests: XCTestCase {
         
         waitForExpectations(timeout: 1.0, handler: nil)
     }
+    
+    func test_BackendServiceErrorInitializable_Init() {
+        
+        let _ = AnyError(BackendServiceError.networkError(.unknownError, nil))
+        
+        XCTAssert(true)
```

Comment:
should test that the object created is not nil

Reply:
The method doesn't return an optional so it either works or it doesn't. I can add a check for nil but it will throw a warning that it will never be nil.

Reply:
sorry, I'm not super familiar with how this works, is there a property or something that you can test that got set properly, due to initialization?

Reply:
adding a check for the errror